### PR TITLE
feat(cmf&container): auto evaluate expression based on props convention

### PR DESCRIPTION
**Pull request by acateland**
**What is the problem this PR is trying to solve?**
only a few selected action properties key could be evaluated.
Meaning each time we need one more we have to touch this package

**What is the chosen solution to this problem?**
Each action properties ending by `Expression` are automaticaly evaluated,  and result put in action propertie missing the `Expression` ending.
`action.stuffExpression` value get evaluated and result put in `action.stuff` and  `action.stuffExpression` get deleted

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/containers/src/actionAPI.js [hunk 1]
```diff
@@ -16,14 +19,13 @@ function evalExpressions(action, context, payload = {}) {
...
-		delete newAction.labelExpression;
-		newAction.label = api.expression.call(action.labelExpression, context, newAction);
-	}
-	if (action.iconExpression) {
-		delete newAction.iconExpression;
-		newAction.icon = api.expression.call(action.iconExpression, context, newAction);
-	}
+	forIn(action, (value, key) => {
+		if (key.match(regexExpression)) {
+			const newKey = key.slice(0, -10);
```
Comment (talend-glorieux) [inline packages/containers/src/actionAPI.js]: Guess you could use a regexp with capture to avoid slicing based on `Expression` length.

Comment (acateland) [reply packages/containers/src/actionAPI.js]: its more safe to

File: packages/containers/src/actionAPI.js [hunk 2]
```diff
@@ -16,14 +19,13 @@ function evalExpressions(action, context, payload = {}) {
...
 	);
-	if (action.labelExpression) {
-		delete newAction.labelExpression;
-		newAction.label = api.expression.call(action.labelExpression, context, newAction);
-	}
-	if (action.iconExpression) {
-		delete newAction.iconExpression;
-		newAction.icon = api.expression.call(action.iconExpression, context, newAction);
-	}
+	forIn(action, (value, key) => {
```
Comment (jmfrancois) [inline packages/containers/src/actionAPI.js]: I prefer Object.keys().forEach which do not need lodash

Comment (acateland) [reply packages/containers/src/actionAPI.js]: we already have loadash on our codebase so.

**Pull request merged**